### ISSUE TITLE
Cannot find models '...ethers/lib/utils'

### ISSUE THREAD

**Issue by backface** [thirdweb-dev/js#1168]
I am building a Nuxt app and everything works as it should in dev mode. However, when generating a signature on the server  in the production build it fails with:

"Cannot find module '/app/.output/server/node_modules/ethers/lib/utils' imported from /app/.output/server/chunks/gen.mjs\nDid you mean to import ethers/lib/utils.js?"

What's wrong here? Could be a library/version mismatch?

**Comment by backface** [comment]
> Hi @backface - Can you create a minimal repo and link it here so we can replicate this issue and fix it ASAP

Here is a nase of a Nuxt starter with signature minting which works for me in dev, but not in production build (I am using pnpm 7.17.0 and node v18.16.0)

https://github.com/backface/thirdweb-nuxt-test

**Comment by MananTank** [comment]
Hi @backface - Looking at the package.json - it seems that you have not installed `ethers@^5`, which is required. Can you add it as a dependency and try again?

**Comment by backface** [comment]
> Hi @backface - Looking at the package.json - it seems that you have not installed `ethers@^5`, which is required.
> 

as a dependency it gets pulled in on install - I have it in version 5.7.2 


> ```
> build: {
> transpile: ['ethers/lib/utils'],
> },
> ```

indeed that seems to work!

> Are you getting this error in dev?

No, not with pnpm, but with yarn I do get it as well

**Comment by MananTank** [comment]
@backface So, does this fix your issue? Can I close this issue?

**Comment by backface** [comment]
@MananTank No. With the transpile option set in now fails in 'run dev' witH

`Caught SyntaxError: The requested module '/_nuxt/node_modules/.pnpm/ethers@5.7.2/node_modules/ethers/lib/utils.js?v=5cdcadb1' does not provide an export named 'isAddress' (at null:33667:10)`

**Comment by joaquim-verges** [comment]
@mahapo @backface fixed it in #1277
